external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -15,6 +15,8 @@
     <NuGetTargetMonikerShort Condition="'$(TargetGroup)' == 'uapaot'">netstandard2.0</NuGetTargetMonikerShort>
     <NuGetTargetMoniker Condition="'$(TargetGroup)' == 'uap'">.NETStandard,Version=v2.0</NuGetTargetMoniker>
     <NuGetTargetMonikerShort Condition="'$(TargetGroup)' == 'uap'">netstandard2.0</NuGetTargetMonikerShort>
+    <!-- Don't warn if some dependencies were rolled forward -->
+    <NoWarn>$(NoWarn);NU1603</NoWarn>
```

Comment:
Can we control the package dependencies to fix this instead of suppressing?

Reply:
Which errors are you getting?  I'm getting the following:

```
F:\git\core-setup\packages\microsoft.dotnet.buildtools\2.0.0-prerelease-01929-02\lib\tool-runtime\project.csproj : error NU1603: Microsoft.Cci 4.0.0-rc4-24217-00 depends on System.Collections.NonGeneric (>= 4.0.1-rc4-24217-00) but System.Collections.NonGeneric 4.0.1-rc4-24217-00 was not found. An approximate best match of System.Collections.NonGeneric 4.0.1-rc4-24217-01 was resolved.
F:\git\core-setup\packages\microsoft.dotnet.buildtools\2.0.0-prerelease-01929-02\lib\tool-runtime\project.csproj : error NU1603: Microsoft.Cci 4.0.0-rc4-24217-00 depends on System.IO.FileSystem.Primitives (>= 4.0.1-rc4-24217-00) but System.IO.FileSystem.Primitives 4.0.1-rc4-24217-00 was not found. An approximate best match of System.IO.FileSystem.Primitives 4.0.1-rc4-24217-01 was resolved.
F:\git\core-setup\packages\microsoft.dotnet.buildtools\2.0.0-prerelease-01929-02\lib\tool-runtime\project.csproj : error NU1603: Microsoft.Cci 4.0.0-rc4-24217-00 depends on System.Collections (>= 4.0.11-rc4-24217-00) but System.Collections 4.0.11-rc4-24217-00 was not found. An approximate best match of System.Collections 4.0.11-rc4-24217-01 was resolved.
F:\git\core-setup\packages\microsoft.dotnet.buildtools\2.0.0-prerelease-01929-02\lib\tool-runtime\project.csproj : error NU1603: Microsoft.Cci 4.0.0-rc4-24217-00 depends on System.Diagnostics.Debug (>= 4.0.11-rc4-24217-00) but System.Diagnostics.Debug 4.0.11-rc4-24217-00 was not found. An approximate best match of System.Diagnostics.Debug 4.0.11-rc4-24217-01 was resolved.
F:\git\core-setup\packages\microsoft.dotnet.buildtools\2.0.0-prerelease-01929-02\lib\tool-runtime\project.csproj : error NU1603: Microsoft.Cci 4.0.0-rc4-24217-00 depends on System.Globalization (>= 4.0.11-rc4-24217-00) but System.Globalization 4.0.11-rc4-24217-00 was not found. An approximate best match of System.Globalization 4.0.11-rc4-24217-01 was resolved.
F:\git\core-setup\packages\microsoft.dotnet.buildtools\2.0.0-prerelease-01929-02\lib\tool-runtime\project.csproj : error NU1603: Microsoft.Cci 4.0.0-rc4-24217-00 depends on System.IO (>= 4.1.0-rc4-24217-00) but System.IO 4.1.0-rc4-24217-00 was not found. An approximate best match of System.IO 4.1.0-rc4-24217-01 was resolved.
F:\git\core-setup\packages\microsoft.dotnet.buildtools\2.0.0-prerelease-01929-02\lib\tool-runtime\project.csproj : error NU1603: Microsoft.Cci 4.0.0-rc4-24217-00 depends on System.IO.FileSystem (>= 4.0.1-rc4-24217-00) but System.IO.FileSystem 4.0.1-rc4-24217-00 was not found. An approximate best match of System.IO.FileSystem 4.0.1-rc4-24217-01 was resolved.
F:\git\core-setup\packages\microsoft.dotnet.buildtools\2.0.0-prerelease-01929-02\lib\tool-runtime\project.csproj : error NU1603: Microsoft.Cci 4.0.0-rc4-24217-00 depends on System.Reflection (>= 4.1.0-rc4-24217-00) but System.Reflection 4.1.0-rc4-24217-00 was not found. An approximate best match of System.Reflection 4.1.0-rc4-24217-01 was resolved.
F:\git\core-setup\packages\microsoft.dotnet.buildtools\2.0.0-prerelease-01929-02\lib\tool-runtime\project.csproj : error NU1603: Microsoft.Cci 4.0.0-rc4-24217-00 depends on System.Resources.ResourceManager (>= 4.0.1-rc4-24217-00) but System.Resources.ResourceManager 4.0.1-rc4-24217-00 was not found. An approximate best match of System.Resources.ResourceManager 4.0.1-rc4-24217-01 was resolved.
F:\git\core-setup\packages\microsoft.dotnet.buildtools\2.0.0-prerelease-01929-02\lib\tool-runtime\project.csproj : error NU1603: Microsoft.Cci 4.0.0-rc4-24217-00 depends on System.Runtime (>= 4.1.0-rc4-24217-00) but System.Runtime 4.1.0-rc4-24217-00 was not found. An approximate best match of System.Runtime 4.1.0-rc4-24217-01 was resolved.
F:\git\core-setup\packages\microsoft.dotnet.buildtools\2.0.0-prerelease-01929-02\lib\tool-runtime\project.csproj : error NU1603: Microsoft.Cci 4.0.0-rc4-24217-00 depends on System.Runtime.Extensions (>= 4.1.0-rc4-24217-00) but System.Runtime.Extensions 4.1.0-rc4-24217-00 was not found. An approximate best match of System.Runtime.Extensions 4.1.0-rc4-24217-01 was resolved.
F:\git\core-setup\packages\microsoft.dotnet.buildtools\2.0.0-prerelease-01929-02\lib\tool-runtime\project.csproj : error NU1603: Microsoft.Cci 4.0.0-rc4-24217-00 depends on System.Runtime.InteropServices (>= 4.1.0-rc4-24217-00) but System.Runtime.InteropServices 4.1.0-rc4-24217-00 was not found. An approximate best match of System.Runtime.InteropServices 4.1.0-rc4-24217-01 was resolved.
F:\git\core-setup\packages\microsoft.dotnet.buildtools\2.0.0-prerelease-01929-02\lib\tool-runtime\project.csproj : error NU1603: Microsoft.Cci 4.0.0-rc4-24217-00 depends on System.Text.Encoding (>= 4.0.11-rc4-24217-00) but System.Text.Encoding 4.0.11-rc4-24217-00 was not found. An approximate best match of System.Text.Encoding 4.0.11-rc4-24217-01 was resolved.
F:\git\core-setup\packages\microsoft.dotnet.buildtools\2.0.0-prerelease-01929-02\lib\tool-runtime\project.csproj : error NU1603: Microsoft.Cci 4.0.0-rc4-24217-00 depends on System.Threading (>= 4.0.11-rc4-24217-00) but System.Threading 4.0.11-rc4-24217-00 was not found. An approximate best match of System.Threading 4.0.11-rc4-24217-01 was resolved.
F:\git\core-setup\packages\microsoft.dotnet.buildtools\2.0.0-prerelease-01929-02\lib\tool-runtime\project.csproj : error NU1701: Package 'Microsoft.Net.Compilers.Targets.NetCore 0.1.5-dev' was restored using '.NETFramework,Version=v4.6.1' instead of the project target framework '.NETCoreApp,Version=v2.0'. This package may not be fully compatible with your project.
```

Where the `Microsoft.Cci` errors are the ones I pinged @weshaggard and @nguerrera about last week.  Did we ever get any resolution on `Microsoft.Cci` getting its dependencies updated?